tests/test_compile_source.py

```diff
@@ -1,14 +1,19 @@
 import pytest
+from packaging.version import Version
 
 import vvm
 
 
-def test_compile_source(foo_source):
+def test_compile_source(foo_source, all_versions):
+    if Version("0.4.0b1") <= all_versions <= Version("0.4.0b5"):
+        raise pytest.skip("vyper 0.4.0b1 to 0.4.0b5 have a bug with combined_json")
```

Comment:
what does `raise pytest.skip` do?

Reply:
skips the test

Reply:
i think `pytest.skip` does that, without the `raise` -- it raises an exception internally

Reply:
https://docs.pytest.org/en/stable/how-to/skipping.html#skipping-test-functions

Reply:
It works without the raise keyword but I find it clearer like this